infer.py

```diff
@@ -90,18 +90,27 @@ def infer():
     # decoders only accept string encoded in utf-8
     vocab_list = [chars.encode("utf-8") for chars in data_generator.vocab_list]
 
-    result_transcripts = ds2_model.infer_batch(
-        infer_data=infer_data,
-        decoding_method=args.decoding_method,
-        beam_alpha=args.alpha,
-        beam_beta=args.beta,
-        beam_size=args.beam_size,
-        cutoff_prob=args.cutoff_prob,
-        cutoff_top_n=args.cutoff_top_n,
-        vocab_list=vocab_list,
-        language_model_path=args.lang_model_path,
-        num_processes=args.num_proc_bsearch,
-        feeding_dict=data_generator.feeding)
+    if args.decoding_method == "ctc_beam_search":
```

Comment:
If `ctc_greedy` doesn't require the external score, I think it's better to move this `if` after line 104

Reply:
Done